### ISSUE TITLE
Monorepo with multiple bs projects

### ISSUE THREAD

**Issue by Coobaha** [reasonml-editor/reasonml-idea-plugin#71]
plugin version: 
0.57
### Description

This is an enhancement request :)
At the moment having one mono-repository opened in IntelliJ with multiple Bucklescript projects won't work. I am not sure if it worth to support but would be nice to have at some point?

**Comment by Coobaha** [comment]
Alternative use case: when Project structure is configured to have multiple content roots

**Comment by giraud** [comment]
I have this problem myself :), and it might be a common use case.
I don't understand alternative use case

**Comment by Coobaha** [comment]
It is possible to configure multiple project roots (not only nested folders in one project) in File -> Project Structure -> Modules.

**Comment by giraud** [comment]
are we talking about this structure ? 
```
  root
  |__ /packages
       |__ /moduleA
       |    |__ bsconfig.json
       |    |__ /lib/js
       |    |__ /src
       |__ /moduleB
            |__ bsconfig.json
            |__ /lib/js
            |__ /src
```

**Comment by Coobaha** [comment]
yes but `./root/packages` is not mandatory and it can be just:
```
    root
       |__ /moduleA
       |    |__ bsconfig.json
       |    |__ /lib/js
       |    |__ /src
       |__ /moduleB
            |__ bsconfig.json
            |__ /lib/js
            |__ /src
```

**Comment by giraud** [comment]
fixed in v0.59
